plugins/bkpr/bookkeeper.c

```diff
@@ -640,7 +640,12 @@ static bool new_missed_channel_account(struct command *cmd,
 				push_credit = AMOUNT_MSAT(0);
 				push_debit = push_amt;
 			} else {
-				ok = amount_msat_sub(&amt, amt, push_amt);
+				if (amount_msat_greater(amt, push_amt)) {
+					ok = amount_msat_sub(&amt, amt, push_amt);
+					assert(ok);
+				} else {
+					amt = AMOUNT_MSAT(0);
+				}
```

Comment:
```suggestion
				} else
					amt = AMOUNT_MSAT(0);
```

Reply:
i see a both of these throughout the repo, when is the latter style preferred? When it's a single statement? (not very used to C)